app/src/vanilla/java/cm/aptoide/pt/app/view/AppViewFragment.java

```diff
@@ -839,7 +839,7 @@ protected void showHideOptionsMenu(@Nullable MenuItem item, boolean visible) {
     List<String> fragmentNames = createFragmentNameList(getFragmentManager().getFragments());
 
     installDisplayable =
-        AppViewInstallDisplayable.newInstance(getApp, installManager, getSearchAdResult(),
+        AppViewInstallDisplayable. newInstance(getApp, installManager, getSearchAdResult(),
```

Comment:
codestyle

Reply:
by the way: this is really weird, I can't find who did this ( a typo I presume) with git blame??? can't understand why aha.